tensorflow/lite/kernels/activations.cc

```diff
@@ -922,11 +954,38 @@ TfLiteStatus PreluEval(TfLiteContext* context, TfLiteNode* node) {
   }
 }
 
+namespace {
+template <typename T>
+void QLeakyRelu(const TfLiteTensor* input, TfLiteTensor* output, float alpha,
+                const LeakyReluOpData* data) {
+  // Quantize the alpha with same zero-point and scale as of input
+  T q_alpha = static_cast<T>(std::max<float>(
```

Comment:
Can we instead do the computation of q_alpha in LeakyReluPrepare and store in the opdata instead of computing this on every Eval?

Reply:
@suharshs , thanks for the review, i have updated the code as per your suggestion, kindly check and approve.

Regards
Amit